### ISSUE TITLE
Process srcset attribute

### ISSUE THREAD

**Issue by tylersticka** [apostrophecms/absolution#5]
This module's great! I use it to prepare content for [my journal's syndication feeds](https://tylersticka.com/journal/feed.json). 

My only complaint so far is that it doesn't appear to process [the `srcset` attribute](https://cloudfour.com/thinks/responsive-images-101-part-4-srcset-width-descriptors/):

```html
<!-- Source markup: -->
<img src="cat.jpg" alt="cat" 
  srcset="cat-320.jpg 320w, cat-640.jpg 640w, cat-1280.jpg 1280w">

<!-- What I would expect: -->
<img src="http://example.com/cat.jpg" alt="cat" 
  srcset="http://example.com/cat-320.jpg 320w, http://example.com/cat-640.jpg 640w, http://example.com/cat-1280.jpg 1280w">

<!-- What happens instead: -->
<img src="http://example.com/cat.jpg" alt="cat" 
  srcset="cat-320.jpg 320w, cat-640.jpg 640w, cat-1280.jpg 1280w">
```

Luckily feed readers don't seem to care too much (it seems like they fall back to `src`), but figured I'd create an issue 'cause sharing is caring. 😁

**Comment by abea** [comment]
Sorry for the delay. We'd be open to reviewing a PR for this.

**Comment by BoDonkey** [comment]
Fixed by PR#15.
